R/mlPrediction.R

```diff
@@ -229,6 +258,16 @@ is.jaspMachineLearning <- function(x) {
   names(model[["tables"]]) <- decodeColNames(names(model[["tables"]]))
   return(model)
 }
+.decodeJaspMLobject.glm <- function(model) {
+  formula <- formula(paste(decodeColNames(as.character(model$terms)[2]), "~", paste0(decodeColNames(strsplit(as.character(model$terms)[3], split = " + ", fixed = TRUE)[[1]]), collapse = " + ")))
```

Comment:
I think this does the same and is slightly easier to parse
```suggestion
  vars <- all.vars(stats::terms(model))
  formula <- formula(paste(
    decodeColNames(vars[1]), 
    "~", 
    paste0(decodeColNames(vars[-1]), collapse = " + ")
  ))
```
Just so you know, this will fail if there are any interaction effects, but so would your original code.

Perhaps the same can be to the `.vglm` method?

Reply:
We can look into simplifying this for all the methods we have now sometime (the vlgm method does this slightly different)! I don't think we include interaction effects in the model in the ml module.